packages/babel-traverse/src/scope/index.js

```diff
@@ -210,7 +210,7 @@ export default class Scope {
    * Generate a unique identifier and add it to the current scope.
    */
 
-  generateDeclaredUidIdentifier(name: string = "temp") {
```

Comment:
Should we mark these `name?: string` so it is clear they are still optional? At a glance, this kind of looks like you've made them required, which isn't the case.

Reply:
Sure thing, kinda forgot this part of the codebase is typed. Gonna fix soon.